modules/platforms/cpp/ignite/odbc/query/data_query.cpp

```diff
@@ -454,16 +441,29 @@ sql_result data_query::process_conversion_result(
     return sql_result::AI_ERROR;
 }
 
-void data_query::set_resultset_meta(const column_meta_vector &value) {
-    m_result_meta.assign(value.begin(), value.end());
+void data_query::set_resultset_meta(column_meta_vector value) {
+    m_result_meta = std::move(value);
     m_result_meta_available = true;
 
     for (size_t i = 0; i < m_result_meta.size(); ++i) {
         column_meta &meta = m_result_meta.at(i);
-        LOG_MSG("\n[" << i << "] SchemaName:     " << meta.get_schema_name() << "\n[" << i << "] TypeName:       "
-                      << meta.get_table_name() << "\n[" << i << "] ColumnName:     " << meta.get_column_name() << "\n["
-                      << i << "] ColumnType:     " << static_cast<int32_t>(meta.get_data_type()));
+        LOG_MSG("[" << i << "] SchemaName: " << meta.get_schema_name());
+        LOG_MSG("[" << i << "] TableName:  " << meta.get_table_name());
+        LOG_MSG("[" << i << "] ColumnName: " << meta.get_column_name());
+        LOG_MSG("[" << i << "] ColumnType: " << static_cast<int32_t>(meta.get_data_type()));
```

Comment:
Does this logging have any performance implications? `LOG_MSG` does not seem to have any checks like `TRACE_MSG` has (`is_trace_enabled`).

Reply:
Logging in ODBC only enabled in debug builds, it's removed altogether in release.